src/plugin_system/plugin_manager.py

```
@@ -358,7 +358,23 @@ def load_plugin(self, plugin_id: str) -> bool:
+            # Register plugin-shipped fonts with the FontManager (if any).
+            # Plugin manifests can declare a "fonts" block that ships custom
+            # fonts with the plugin; FontManager.register_plugin_fonts handles
+            # the actual loading. Wired here so manifest declarations take
+            # effect without requiring plugin code changes.
+            font_manifest = manifest.get('fonts')
+            if font_manifest and self.display_manager is not None:
+                font_manager = getattr(self.display_manager, 'font_manager', None)
+                if font_manager is not None and hasattr(font_manager, 'register_plugin_fonts'):
+                    try:
+                        font_manager.register_plugin_fonts(plugin_id, font_manifest)
+                    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)